utilities/upgrade-service/install-upgrade.sh

```diff
@@ -71,13 +124,14 @@ fi
 
 echo "Upgrading from $PREV_IMAGE to $IMG";
 
-mkdir -p /tmp/managed-gitops
-cd /tmp/managed-gitops
+# create the required yaml files for the kustomize based install.
+create_kustomization_init_file
 
-git clone git@github.com:redhat-appstudio/managed-gitops.git 
-cd managed-gitops
+# Set the right container image and apply the manifests
+${KUSTOMIZE} build ${TEMP_DIR} |  COMMON_IMAGE=${IMG} envsubst | kubectl apply -f -
 
-make install-all-k8s IMG=$IMG
```

Comment:
One reason to use `make install-all-k8s` was that it executes db-migrate as one of the steps making sure the changes in database are migrated to the new version. I don't think applying manifests will help perform this actions.

Reply:
This is no longer required, as the `backend` component on startup  calls `db.Migrate()` to create the required DB schema. https://github.com/redhat-appstudio/managed-gitops/blob/911d13321ed6217b005562ec0e37b4b275f82cd5/backend/main.go#L92